app/services/site_setting/update.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class SiteSetting::Update
```

Comment:
Renamed `UpdateSiteSetting` to `SiteSetting::Update` to follow best practices from the upcoming docs.